solution.rb

```diff
@@ -0,0 +1,19 @@
+count = {}
```

Comment:
If you instantiate this hash with a default value of zero (`count = Hash.new(0)`), you don't need to check if a given key is in the hash before incrementing its value.